<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>2</number>
     </property>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>生日</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>婚否</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>住址</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>梁山伯</string>
      </property>
      <property name="icon">
       <iconset resource="photos.qrc">
        <normaloff>:/images/butterfly01.png</normaloff>:/images/butterfly01.png</iconset>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>300/3/3</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>梁家村</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>祝英台</string>
      </property>
      <property name="icon">
       <iconset resource="photos.qrc">
        <normaloff>:/images/butterfly02.png</normaloff>:/images/butterfly02.png</iconset>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>女</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>301/4/4</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>祝家村</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>头像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPhotos">
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/images/butterfly01.png</normaloff>:/images/butterfly01.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/images/butterfly02.png</normaloff>:/images/butterfly02.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/images/cute01.png</normaloff>:/images/cute01.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/images/cute02.png</normaloff>:/images/cute02.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photos.qrc">
          <normaloff>:/images/toofat.png</normaloff>:/images/toofat.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGender"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>生日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxIsMarried">
       <property name="text">
        <string>婚否</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>住址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDel">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="photos.qrc"/>
 </resources>
 <connections/>
</ui>
